Source/ToolbarAction.m

```diff
@@ -0,0 +1,35 @@
+//
+//  ToolbarAction.m
+//  Pods
+//
+//  Created by Matt Bush on 8/29/16.
+//
+//
+
+#import "ToolbarAction.h"
+
+@implementation ToolbarAction
+
+- (void)setBackgroundImage:(NSString *)imageName
+{
+    [self.imageView setContentMode: UIViewContentModeScaleAspectFill];
```

Comment:
Double checking to make sure you want this to be aspect fill? Images will most likely be the correct proportion anyway.